ccp/index_reader.py

```diff
@@ -188,7 +184,7 @@ def read_yaml(self, filepath):
             with open(filepath) as fin:
                 data = yaml.load(fin, Loader=yaml.BaseLoader)
         except yaml.YAMLError as exc:
-            print ("Failed to read {}".format(filepath))
+            _print("Failed to read {}".format(filepath))
             raise(exc)
         else:
```

Comment:
how are we handling this `raise` here we are raising exception from exception?

Reply:
We are raising the exception but ensuring we print error message along with filepath with issue.

Reply:
We have not handled this. Index Reader expects the yaml files to be in sane format. Do we want to log error and proceed further in this case as well?

Reply:
yeah, these should be taken care of by the index CI. But index reader should not fail entirely for single malformed entry. I would suggest we log it and continue to process other entries.

Reply:
Agreed. I'll update the PR.